docs/building-a-form/available-form-features-and-usage-guidelines.md

```diff
@@ -414,6 +414,8 @@ These properties are nested under `uiSchema: { `ui:options`: {} }`:
 - `hideOnReview`: Hides the field on the review page; takes a `boolean`
 - `hideOnReviewIfFalse`: Hides the field on the review page when the field value is `false`; takes a `boolean`
 - `keepInPageOnReview`: By default, array fields that are displayed on a single page in a form, such as information for multiple dependents, are displayed in a separate section on the review page. To keep the information in a single section on a review page, set this property.
+- `reviewEditButtonAriaLabel`: By default, the Edit button for pages on the review page will have an `aria-label` value of
```

Comment:
Hm, not sure I can do any better. Perhaps `ariaLabelForEditButtonOnReview`? I know it's even longer, but I wonder if it reads a bit more clearly by leading with what it is and then describing the context? Starting the name with `review` might be confusing, as it's referencing a noun, but could be read as a verb. But I don't have strong feelings about this.

The only thing I might add here is a bit more context for why this exists. I assume it's because there can be multiple edit buttons on the review page, and this helps distinguish which button is which?

Reply:
Yeah, I can add some more context. I think it's more that "Edit <page title>" might not always make sense and this gives people the option to customize it in cases where the default is confusing.

Reply:
Got it, makes sense. Thanks for pushing an update for this!